extensions/indexes/range/src/main/java/org/exist/indexing/range/RangeIndexConfigAttributeCondition.java

```diff
@@ -94,10 +95,9 @@ public RangeIndexConfigAttributeCondition(final Element elem, final NodePath par
             operator = Operator.EQ;
         }
 
+        // As default the range index shall be case-sensitive, unless explicitly set to 'no'.
         final String caseString = elem.getAttribute("case");
-
-        // Todo: discuss and agree
-        caseSensitive = (caseString != null && !"no".equalsIgnoreCase(caseString));
+        caseSensitive = !StringUtils.equalsIgnoreCase(caseString, "no");
```

Comment:
whats the difference to this??
```suggestion
        caseSensitive = !"no".equalsIgnoreCase(caseString);
```
Results this way:
```
jshell> !"no".equalsIgnoreCase(null);
$1 ==> true

jshell> !"no".equalsIgnoreCase("");
$2 ==> true

jshell> !"no".equalsIgnoreCase("yes");
$3 ==> true

jshell> !"no".equalsIgnoreCase("No");
$4 ==> false

jshell> !"no".equalsIgnoreCase("no");
$5 ==> false
```

Reply:
I also was expecting this solution. (@reinhapa I think I have been looking for something like `jshell` for a while now)

Reply:
It also leads to dependency problems in maven build ...
https://github.com/eXist-db/exist/actions/runs/13908483284/job/38916986472?pr=5670#step:6:8842